sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/ServiceBusEndToEndTests.cs

```diff
@@ -743,6 +743,19 @@ public class ServiceBusMultipleMessagesTestJob_BindToMessageArray
                 string[] messages = array.Select(x => x.Body.ToString()).ToArray();
                 ServiceBusMultipleTestJobsBase.ProcessMessages(messages);
             }
+
+            public static async void SBQueue2SBQueueAutoCompleteDisabled(
+                [ServiceBusTrigger(SecondQueueNameKey, AutoCompleteMessages = false)]
```

Comment:
Will this actually be executed? I don't see any messages being send to the second queue (I think the function is probably named incorrectly).